xml/admin_ceph_troubleshooting.xml

```diff
@@ -518,4 +518,35 @@ keep_jobs: 1</screen>
    </procedure>
   </tip>
  </sect1>
+ <sect1 xml:id="trouble-osd-panic">
+  <title>OSD Panic Occurs when Media Error Happens during &filestore; Directory Split</title>
+
+  <para>
+   When directory split error is generated on &filestore; OSDs, the
```

Comment:
This is missing an article. Either:

When *a* directory split error is generated on *a* &filestore; OSD, the...

... or...

When directory split errors are generated on &filestore; OSDs, the...

I slightly prefer the second, as the sentence continues that _the corresponding_ OSD, implying that the one that threw the error terminates. If the sentence is in the singular, then there is no group for there to be a "corresponding" one of.

Reply:
It should be:

When a directory split error is generated on a &filestore; OSD, the...

Reply:
fixed in c1581590